src/pages/index.js

```diff
@@ -546,9 +560,17 @@ const Index = ({ data: fallbackData, userAgent }) => {
       setShouldShowSfacExtensionPrompt(
         !searchbarSfacPrompt && showSfacExtensionPrompt
       )
-      setShouldShowSearchbarSFACPrompt(searchbarSfacPrompt)
+      setShouldShowSearchbarSFACPrompt(
+        searchbarSfacPrompt && !(impactType === CAUSE_IMPACT_TYPES.group)
```

Comment:
Will we just never show this version of the SFAC prompt when group impact is enabled? That's probably OK temporarily, but we'll want to figure out the way forward on this. This version of the SFAC notification is not a conclusive winner on the experiment.

Reply:
I'm going to file a ticket so we don't lose this context! TFAC-1217